tetrateci/setup_boring_go.sh

```diff
@@ -1,13 +1,28 @@
-#!/bin/bash
+#!env bash
+set -e
+
+if $(grep "1.7" <<< $TAG ); then
+  export GOLANG_VERSION=1.14.12b4
```

Comment:
Thinking loud, I think it is better that we refrain from mentioning the patch version.

Reply:
Have to look at this, right now I don't have much idea how to work around that